x-pack/plugins/fleet/server/types/models/agent_policy.ts

```diff
@@ -47,7 +47,6 @@ export const AgentPolicyBaseSchema = {
       })
     )
   ),
-  is_protected: schema.maybe(schema.boolean()),
```

Comment:
isn't it a breaking change to remove a field?

Reply:
this feature isn't released yet, so this shouldn't be a problem